api/naming/naming.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/docker/swarmkit/api"
 )
 
+const (
+	// PredefinedLabel identifies internally allocated swarm resources
+	PredefinedLabel = "com.docker.swarm.predefined"
```

Comment:
@aboch At this time, we have no other labels in this package. Let's avoid creating that dependency without ensuring it is the right way to go. In a later PR, consolidating seems reasonable, but for now, just copy paste the string.